example/vite.config.ts

```diff
@@ -10,7 +10,17 @@ export default defineConfig(() => {
       // enables debugging in workbox
       "process.env.NODE_ENV": JSON.stringify("development"),
     },
-    plugins: [qwikCity(), qwikVite(), tsconfigPaths(), qwikPwa()],
+    plugins: [
+      qwikCity(),
+      qwikVite(),
+      tsconfigPaths(),
+      qwikPwa({
+        assets: {
```

Comment:
by default, the user should just bring this in the config without any option, by default everything should be handled, so let's remove the options object here so we simplify the example and the usage for users.